mathesar_ui/src/sections/table-view/header/HeaderCell.svelte

```diff
@@ -35,6 +49,22 @@
     } else {
       meta.addGroup(column.name);
     }
+    closeMenu();
+  }
+
+  async function toggleAllowDuplicates() {
+    isRequestingToggleAllowDuplicates = true;
+    try {
+      const isUnique = await constraintsDataStore.updateUniquenessOfColumn(column, (u) => !u);
+      const msg = `Column "${column.name}" will ${isUnique ? 'no longer ' : ''}allow duplicates.`;
+      console.log(msg); // TODO display success toast message: msg
```

Comment:
We could add a eslint ignore before the console.log line to avoid eslint warnings.

Reply:
Addressed in 7d3ba58d38a291cabed95322d59bbd8e26532c8b